src/components/manage/utils.js

```diff
@@ -325,9 +325,14 @@ export const processTier = (crowdsaleAddress, crowdsaleNum) => {
     })
 }
 
-const setCurrentContract = contract => {
+const setCurrentContract = (contract, index) => {
   const currentDate = new Date().getTime()
-  if (contract.startDate <= currentDate && currentDate <= contract.endDate) contract.current = true
+  const { startDate, endDate } = contract
+
+  if (index === 0 && currentDate < startDate || startDate <= currentDate && currentDate <= endDate) {
```

Comment:
This condition is a little hard to read. At the very least we should use parentheses to make the precedence clear:

```
if ((index === 0 && currentDate < startDate) || (startDate <= currentDate && currentDate <= endDate))
```

Even better, extract the sub-conditions to variables:

```
const isActive = startDate <= currentDate && currentDate <= endDate
const isFirstTier = index === 0

if (isActive || (isFirstTier && currentDate < startDate)
```

Reply:
updated, thanks!